### DISCUSSION TITLE
Issue with tab navigation

### DISCUSSION THREAD

**Discussion by emmanuelxd** [megahertz/react-simple-wysiwyg#49] [Q&A]
Hello React Simple WYSIWYG team,

I hope you're doing well. I'm encountering an issue related to tab navigation in the editor and would appreciate your guidance or suggestions.

Detailed Description:

Currently, I'm using React Simple WYSIWYG to integrate a rich text editor into my application. However, when using div elements with contenteditable="true", I'm facing challenges with tab navigation. Specifically, I'm unable to make these div elements focusable and navigable using the Tab key.

**Comment by megahertz** [comment]
Yes, it doesn't handle tab navigation properly. Fixed in 3.1.1.

**Reply by emmanuelxd** [reply]
Thank you sir!
